### PR TITLE
MRC-175 Screen reader content

### PR DESCRIPTION
**Pull request by adamjw3**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -202,14 +204,16 @@ fullResultsPage.taxableProfit = Taxable Profit
...
 
 
 associatedCompanies.title = Number of associated companies
 associatedCompanies.heading = Did your company have any active associated companies?
 associatedCompanies.heading.between = Did your company have any active associated companies between {0} and {1}?
 associatedCompanies.warning = Do not include the company you''re currently calculating Marginal Relief for in your answer. Doing so will provide you with an incorrect result.
 associatedCompanies.detailsTitle = What are associated companies?
-associatedCompanies.detailsHtml = <p>A company is considered associated to another if:</p><ul class="govuk-list govuk-list--bullet"><li>one of the companies has <a href="https://www.gov.uk/hmrc-internal-manuals/company-taxation-manual/ctm60210" class="govuk-link" target="_blank">control (opens in new tab)</a> of the other</li><li>both companies are controlled by the same companies or people</li></ul>
+associatedCompanies.detailsHtml = <p>A company is considered associated to another if:</p><ul class="govuk-list govuk-list--bullet"><li>one of the companies has control of the other</li><li>both companies are controlled by the same companies or people</li></ul>
```
Comment (girishkamat) [inline conf/messages.en:216]: Is this a mistake?

File: app/views/ResultsPageView.scala.html [hunk 2]
```diff
@@ -33,7 +33,7 @@
 
 @layout(pageTitle = titleNoForm(messages("resultsPage.title"))) {
 
-    @displayBanner(calculatorResult)
+    @addBannerScreenReader(displayBanner(calculatorResult), calculatorResult)
```
Comment (girishkamat) [inline app/views/ResultsPageView.scala.html:36]: Why dont we incorporate this functionality into the displayBanner method? Looks like every banner needs a bit of text for screen reader

File: app/views/ResultsPageView.scala.html [hunk 3]
```diff
@@ -46,7 +46,7 @@ <h2 class="govuk-heading-m" style="margin-bottom: 7px;">@messages("resultsPage.c
     }
 
     <div class="app-table">
-        @{replaceTableHeader(displayCorporationTaxTable(calculatorResult))}
+        @{replaceTableHeader(messages("resultsPage.corporationTaxTableSummary"),displayCorporationTaxTable(calculatorResult))}
```
Comment (girishkamat) [inline app/views/ResultsPageView.scala.html:49]: corporationTaxTableSummary - can we more clear that its screen reader summary i.e corporationTaxTableScreenReaderSummary ?

Comment (girishkamat) [reply app/views/ResultsPageView.scala.html:49]: Also, cant we move the replacement to the displayCorporationTaxTable method itself. Also applies to displayEffectiveTaxTable

File: conf/messages.en (hunk 1 above)
Comment (adamjw3) [reply conf/messages.en:220]: No this is from the changes in access ticket, one of the tasks was to remove the link, but i also removed the word "control", so just adding that back in

File: app/views/helpers/FullResultsPageHelper.scala [hunk 4]
```diff
@@ -389,9 +392,18 @@ object FullResultsPageHelper extends ViewHelper {
           rows = Seq(
             Seq(
               TableRow(content = HtmlContent(p(messages("fullResultsPage.taxableProfit.daysAllocated")))),
-              TableRow(content = HtmlContent(d1.days.toString), classes = "govuk-table__cell--numeric"),
-              TableRow(content = HtmlContent(d2.days.toString), classes = "govuk-table__cell--numeric"),
-              TableRow(content = HtmlContent(totalDays.toString), classes = "govuk-table__cell--numeric")
+              TableRow(
+                content = HtmlContent(s"""${d1.days.toString} <span class="sr-only">${messages("resultsPage.days")}</span>"""),
```
Comment (girishkamat) [inline app/views/helpers/FullResultsPageHelper.scala:396]: The span code for screen reader is duplicated a lot. Can we not add an implicit class to simplify this?

**Comment by girishkamat** [comment]
Dont think any of my comments have been addressed

**Review by girishkamat — APPROVED** [review]

**Pull request merged**